### ISSUE TITLE
Make it clearer that staking is only availalbe on the ETH network

### ISSUE THREAD

**Issue by helpfulBro** [aave/interface#697]
**Is your feature request related to a problem? Please describe.**
When a user visits the stake menu and is not on the ETH network the UI elements are disabled but there is no copy advising how they can stake.

**Describe the solution you'd like**
Include some copy and perhaps a CTA button allowing the user to switch to the Ethereum network to interact with the staking page.

**Describe alternatives you've considered**
None

**Additional context**
Image of disabled stake page when not on ETH network
![image](https://user-images.githubusercontent.com/25915621/164567808-9cf57098-3b22-44aa-a1fd-c6f3dc422171.png)

**Comment by iamanastasia** [comment]
Quick questions/comments to define problem more accurately. 

1. Probably 'Stake' buttons are disabled because there is 0 AAVE/ABPT balance in the wallet. Here are my screenshots where there is AAVE balance and no ABPT balance. 
![step 1](https://user-images.githubusercontent.com/92739089/170474173-e49af013-c78e-40e6-9cca-0d9acace6a67.png)

**Solution suggestion #1:** Add AAVE/ABPT wallet balance and disable 'Stake' button when balance = 0. 
![Ethereum network](https://user-images.githubusercontent.com/92739089/170474407-0096d4c8-1e0f-4424-96dc-93ab7f5b015a.png)


2. Staking shows Mainnet info and suggests to switch Network only in tx modal window, which is not very friendly if you have 0 balance and can't open any tx window. (screenshot proof) 
![step 2](https://user-images.githubusercontent.com/92739089/170474258-cf3d912d-49e2-4f03-b3bc-be0e850ec064.png)

**Solution suggestion #2:** Add 1 more place to show current network and suggestion to switch it.
![Wrong network v1](https://user-images.githubusercontent.com/92739089/170474462-2fe11d7a-6d0d-4e45-afc3-a81f3d65336c.png)


3. There is Testnet issue. Whenever testnet switcher is enabled App shows Kovan info. And I see 2 problems here: a) It's not obvious you are in the testnet (where is my balance?!) b) no identification that it should be Kovan network. 

**Solution suggestion #3:** Add big non-dismissable banner on top across all sections whenever testnet switch is turned on. 
![Staking on Testnet](https://user-images.githubusercontent.com/92739089/170474539-8c832a2e-4940-45ae-a0de-a922a359152e.png)

**Solution suggestion #4:** Add 'Switch to Kovan' banner for Staking (should user be able to see mainnet info like he does in the Dashboard?)
![Testnet wrong network](https://user-images.githubusercontent.com/92739089/170474613-e2d145a7-8c65-4b1a-9082-a9f715a03340.png)


Let me know what you think!

**Comment by helpfulBro** [comment]
I think all of these suggestions are great.

One other point that many users ask about is how they can get the BPT token, today it's almost hidden in the 'Balancer liquidity pool'  link below the staking panel.

How can we make it easier for users to find the correct pool on balance when their token balance = 0?

**Comment by iamanastasia** [comment]
@helpfulBro looks like BPT deserves a separate issue :D though, can you explain more? 
1. Are there requests coming for this?
2. When user has 0 AAVE balance, should we also provide info how to get it? 

------

@sakulstra @sendra @defispartan Any thoughts on designs above?

**Comment by helpfulBro** [comment]
> @helpfulBro looks like BPT deserves a separate issue :D though, can you explain more?
> 
>     1. Are there requests coming for this?
> 
>     2. When user has 0 AAVE balance, should we also provide info how to get it?

1. Yep we get a few questions each week asking how to get the BPT
2. Yes I think this would be helpful
3. BPT issue : https://github.com/aave/interface/issues/817

Thanks!

**Comment by foodaka** [comment]
@iamanastasia working on this here can modify it to your design specifications

![Screenshot 2022-06-08 at 13 23 10](https://user-images.githubusercontent.com/8342048/172615216-41979556-0c75-4b78-a622-c7e1e3a0823e.png)

**Comment by iamanastasia** [comment]
Hey @foodaka Could we make the banner dark-mode style? 
<img width="923" alt="Screenshot 2022-06-08 at 16 28 48" src="https://user-images.githubusercontent.com/92739089/172616322-8bcd9df8-b332-4915-aa3c-7cab3ccf6c70.png">

**Comment by defispartan** [comment]
@iamanastasia Is there a design for this task? 

There were other prs which made changes to staking but I don't think any have addressed this.

**Comment by iamanastasia** [comment]
@defispartan Yes there are designs (warnings), @foodaka I guess you were involved into building this, do you have any updates on this?

**Comment by foodaka** [comment]
There is a PR linked here that i had started but i was waiting for other staking dependencies to finish. I can pick it up again if the designs are finalized :)

**Comment by iamanastasia** [comment]
Hi, everyone. 
So there were several updates on this page regarding initial problem that was described.
1. User can see his staking balance at any network. 
2. We ask user to switch network when they want to interact with their balance (screenshot below).
3. 'GET ABPT' token button is somehow emphasised and explained in a more obvious manner.

Another update that might come soon is showing wallet balance (Stake button is disabled when balance = 0 and without wallet balance it's not obvious). 

So @helpfulBro @foodaka Do you think with those changes we can  close this task and consider it to be done?

**Comment by helpfulBro** [comment]
I think the banner is still beneficial, or at least update the copy to something like

"AAVE holders can stake their AAVE in the Safety Module (**Ethereum Network only**) to add more security to the protocol and earn Safety Incentives. 

![image](https://user-images.githubusercontent.com/25915621/193399490-d9c6e33c-2e09-4f1d-bc70-91449171c5de.png)
